.update-copyright.conf

```diff
@@ -0,0 +1,17 @@
+[project]
+name: lesson-template
+vcs: Git
+
+[files]
+authors: yes
+files: no
+
+[copyright]
+short: {project} comes with ABSOLUTELY NO WARRANTY and is licensed under the MIT and CC BY 4.0 licenses.
+long: This file is part of {project}.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
On Sun, Mar 15, 2015 at 07:10:58PM -0700, Greg Wilson wrote:
> Doesn't this all duplicate what's in the LICENSE file?

Yup.  But I like the explicitness of per-file copyright blurbs (which
is really what #198 is about).  For other advice along this lines, the
FSF says (about the GPL) [1]:

  Whichever license you plan to use, the process involves adding two
  elements to each source file of your program: a copyright notice
  (such as “Copyright 1999 Terry Jones”), and a statement of copying
  permission, saying that the program is distributed under the terms
  of the GNU General Public License (or the Lesser GPL).

For more details motivating this approach, see [2].

For folks on both sides of per-file blurbs, see [3].

[1]: http://www.gnu.org/licenses/gpl-howto.html
[2]: http://lu.is/blog/2012/03/17/on-the-importance-of-per-file-license-information/
[3]: http://programmers.stackexchange.com/questions/125836/do-you-have-to-include-a-license-notice-with-every-source-file